postgres-driver/load_balancer.go

```
@@ -34,19 +49,31 @@ func (lb *SelectLoadBalancersRow) toLoadBalancer() *types.LoadBalancer {
 		GigastakeRedirect: lb.GigastakeRedirect.Bool,
 
 		StickyOptions: types.StickyOptions{
-			Duration:      lb.Duration.String,
-			StickyOrigins: lb.Origins,
-			StickyMax:     int(lb.StickyMax.Int32),
-			Stickiness:    lb.Stickiness.Bool,
+			Duration:      lb.SDuration.String,
+			StickyOrigins: lb.SOrigins,
+			StickyMax:     int(lb.SStickyMax.Int32),
+			Stickiness:    lb.SStickiness.Bool,
 		},
 
 		CreatedAt: lb.CreatedAt.Time,
 		UpdatedAt: lb.UpdatedAt.Time,
 	}
+
+	// Unmarshal LoadBalancer Users JSON into []types.UserAccess
+	err := json.Unmarshal(lb.Users, &loadBalancer.Users)
+	if err != nil {
```

This can silently fail. It will trigger a further error if `lb.Users` is empty?

---

If `lb.Users` is empty it just ends up with an empty Users slice; there's no error.

That said, in the driver code that creates a `LoadBalancer`, there is a check that throws an error if there are no users in the input so it shouldn't be possible for a `LoadBalancer` to exist without at least one User.

And for existing LBs, when we are ready to merge this code we need to run a script to create a user with the `OWNER` role for all current users.